src/components/Login.js

```diff
@@ -9,35 +9,32 @@ class Login extends Component {
       password: ''
     };
   }
+
   handleChange(event) {
     this.setState({
       [event.target.name]: event.target.value
     })
     event.preventDefault();
   }
-  handleSubmit(event) {
-     // Connect to backend
-     alert(`E-mail: ${this.state.email}, Password: ${this.state.password}`);
-    event.preventDefault();
-  }
+
   render() {
     return (
       <div className="container">
         <h3>Login to...</h3>
         <div className="row">
-          <form className="col s12" onSubmit={event => this.handleSubmit(event)}>
+          <form className="col s12" onSubmit={(event) => this.props.onSubmit(event,this.state)}>
```

Comment:
By passing it on like this, the enclosing Component needs to know that it needs to do the `event.preventDefault()`. It would be better to keep this inside the the Login Form.